changelog.rst

```diff
@@ -1,12 +1,132 @@
+The support for Numpy 1.20 that was first added in QuTiP 4.5.3 is present in this version as well, and the same build considerations mentioned there apply here too.
+If building using the now-supported PEP 517 mechanisms (e.g. ``python -mbuild /path/to/qutip``), all build dependencies will be correctly satisfied.
+
+Improvements
+------------
+- **MAJOR** Add saving, loading and resetting functionality to ``qutip.settings`` for easy re-configuration. (by **Eric Giguère**)
+- **MAJOR** Add a quantum gate scheduler in ``qutip.qip.scheduler``, to help parallelise the operations of quantum gates.  This supports two scheduling modes: as late as possible, and as soon as possible. (by **Boxi Li**)
+- **MAJOR** Improved qubit circuit simulators, including OpenQASM support and performance optimisations. (by **Sidhant Saraogi**)
+- **MAJOR** Add tools quantum measurements and their statistics. (by **Simon Cross** and **Sidhant Saraogi**)
+- Add support for Numpy 1.20.  QuTiP should be compiled against a version of Numpy ``>= 1.16.6`` and ``< 1.20`` (note: does _not_ include 1.20 itself), but such an installation is compatible with any modern version of Numpy.  Source installations from ``pip`` understand this constraint.
+- Improve the error message when circuit plotting fails. (by **Boxi Li**)
+- Add support for parsing M1 Mac hardware information. (by **Xiaoliang Wu**)
+- Support decomposition of ``X``, ``Y`` and ``Z`` gates in circuits. (by **Boxi Li**)
+
+Bug Fixes
+---------
+- Fix ``dims`` in the returns from ``Qobj.eigenstates`` on superoperators. (by **Jake Lishman**)
+- Calling Numpy ufuncs on ``Qobj`` will now correctly raise a ``TypeError`` rather than returning a nonsense ``ndarray``. (by **Jake Lishman**)
+- Convert segfault into Python exception when creating too-large tensor products. (by **Jake Lishman**)
+- Correctly set ``num_collapse`` in the output of ``mesolve``. (by **Jake Lishman**)
+- Fix ``ptrace`` when all subspaces are being kept, or the subspaces are passed in order. (by **Jake Lishman**)
+- Fix sorting bug in ``Bloch3d.add_points()``. (by **pschindler**)
+- Fix invalid string literals in docstrings and some unclosed files. (by **Élie Gouzien**)
+- Fix Hermicity tests for matrices with values that are within the tolerance of 0. (by **Jake Lishman**)
+- Fix the trace norm being incorrectly reported as 0 for small matrices. (by **Jake Lishman**)
+- Fix issues with ``dnorm`` when using CVXPy 1.1 with sparse matrices. (by **Felipe Bivort Haiek**)
+- Fix segfaults in ``mesolve`` when passed a bad initial ``Qobj`` as the state. (by **Jake Lishman**)
+- Fix sparse matrix construction in PIQS when using Scipy 1.6.1. (by **Drew Parsons**)
+- Fix ``zspmv_openmp.cpp`` missing from the pip sdist. (by **Christoph Gohlke**)
+- Fix correlation functions throwing away imaginary components. (by **Asier Galicia Martinez**)
```

Comment:
https://github.com/qutip/qutip/pull/1269 and https://github.com/qutip/qutip/pull/1194
```suggestion
- Fix correlation functions throwing away imaginary components. (by **Asier Galicia Martinez**)
- Fix ``QubitCircuit.add_circuit()`` for SWAP gate. (by **Canoming**)
- Fix the brocken LaTeX image conversion. (by **Jake Lishman**)
```